backend/coins/btc/electrum/electrum.go

```diff
@@ -216,7 +226,7 @@ func CheckElectrumServer(serverInfo *config.ServerInfo, log *logrus.Entry, diale
 	// We receive the first one that comes back.
 	defer electrumClient.Close()
 	go func() {
-		_, err := electrumClient.ServerVersion()
+		_, err := electrumClient.ServerBanner()
```

Comment:
what about using the `server.ping` method instead? the server banner has a bit of a bad history, as servers return phishing text there.

Reply:
:+1: done